odl/solvers/advanced/proximal_operators.py

```diff
@@ -258,90 +470,242 @@ def __init__(self, sigma):
             super().__init__(domain=space, range=space, linear=False)
 
         def _call(self, x, out):
+            """Apply the operator to ``x`` and stores the result in ``out``."""
+
+            if g is None:
+                step = self.sigma * lam / x.norm()
+
+                if step < 1.0:
+                    out.lincomb(1.0 - step, x)
+                else:
+                    out.set_zero()
+
+            else:
+                step = self.sigma * lam / (x - g).norm()
+
+                if step < 1.0:
+                    out.lincomb(1.0 - step, x, step, g)
+                else:
+                    out.assign(g)
+
+    return ProximalL2
+
+
+def proximal_cconj_l2_squared(space, lam=1, g=None):
+    """Proximal operator factory of the convex conj of the squared l2-norm.
+
+    Function for the proximal operator of the convex conjugate of the
+    functional F where F is the l2-norm
```

Comment:
We can safely drop it I guess. As long as there is a simple possibility to apply this scaling to the proximal if the user wants it.